### PR TITLE
chats are now loaded from daemon api

### PR DESCRIPTION
**Pull request by broneks**
Although this works and connects to the API, we will need to seed some dummy data on the API end in order to keep developing operator. Spinning up the example app and writing some messages isn't enough because certain fields are blank (e.g. first name and last name).

I spent some time on it yesterday but didn't end up with much. I'll try again later.

### CODE REVIEW

File: src/store/Chat/reducer.js [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
   CHAT_SET_ACTIVE,
   CHAT_TOGGLE_OPEN,
   OPERATOR_SET_FILTER,
 } from './constants';
 
-import makeDummy from '../dummy';
-
-
-// Flag to enable putting dummy data into redux
-const DUMMY_DATA = true;
```
Comment (mihok) [inline src/store/Chat/reducer.js]: Should we pull this from an environment variable and then when we're just working on the Operator without running the Daemon we can use dummy data, but if we're working on a Daemon+Operator task we can disable it and load from the Daemon? I'm just thinking from a dependencies perspective

File: src/store/endpoints.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const DAEMON_HOST = process.env.MNML_DAEMON_HOST || 'http://localhost:8000';
```
Comment (mihok) [inline src/store/endpoints.js]: This will come from Redux state config object I added in #40

**Comment by broneks** [comment]
Ok I will update so that I'm using the state config.

**Comment by broneks** [comment]
PR updated.

I removed the API host env variable and set up the action to eventually get the url from the config on state. It feels weird passing state into an action creator, but I don't know how else you would do it.

Also should the config be inside the "chats" reducer or should we extract it into a "settings" reducer?

**Comment by teesloane** [comment]
If config _isn't_ changing (I can't remember) it should not be in a reducer at all.

**Comment by broneks** [comment]
Is this PR ok? Is there anything that I should change?

**Comment by mihok** [comment]
I think this looks great, apologizes for the delay!

**Pull request merged**